### PR TITLE
Update ci config and add Dockerfile for running App in local

### PR DESCRIPTION
**Pull request by ganezasan**
Hi,  I added some changes for improving this project. 

- update circleci config
  -  add workflow
  -  use Anchor/Alias of YAML function
- add Dockerfile for running App in local
- change the root path in App
- update ruby version 2.4.2 to 2.6.3 
- update rails version 5.1.4 to 5.2.3

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -1,44 +1,54 @@
...
+      - image: circleci/postgres:<< parameters.postgres_tag >>
         environment:
           POSTGRES_USER: circleci-demo-ruby
           POSTGRES_DB: rails_blog
           POSTGRES_PASSWORD: ""
+
+jobs:
+  build:
+    docker:
+      - image: circleci/ruby:2.4.2-jessie-node
```
Comment (lokst) [inline .circleci/config.yml]: Would it be good to use a newer version of Ruby that is also Stretch-based?

File: .circleci/config.yml [hunk 2]
```diff
@@ -55,26 +65,36 @@ jobs:
...
+            - rails-demo-yarn-
       - run:
           name: Wait for DB
           command: dockerize -wait tcp://localhost:5432 -timeout 1m
-
       - run:
           name: Database setup
-          command: bin/rails db:schema:load --trace
-
+          command: bundle exec rails db:schema:load --trace
```
Comment (lokst) [inline .circleci/config.yml:80]: I like that we are using `bundle exec`!

**Comment by lokst** [comment]
It might be good to update the year in `Copyright (c) 2017 CircleCI` too

File: .circleci/config.yml (hunk 1 above)
Comment (ganezasan) [reply .circleci/config.yml]: @lokst Thanks, it sounds great! I updated the ruby version to 2.6.3 and use Stretch-based image.

> https://github.com/CircleCI-Public/circleci-demo-ruby-rails/pull/17/commits/d98f9e04f03e7bc6a411df005dadea0e2317f27e

**Comment by ganezasan** [comment]
@lokst Thanks! I update the Copyright to 2019 :)
@BoVice I updated the config to 2.0. And added some fixes. could you review it?

**Review by jasonsouza — APPROVED** [review]
Looks good! Thanks @ganezasan !

**Pull request merged**